test/e2e/node_feature_discovery_test.go

```diff
@@ -772,6 +772,143 @@ core:
 				})
 			})
 		})
+
+		Context("Deploy single nfd-master pod with config file", Ordered, func() {
+			var (
+				crds          []*apiextensionsv1.CustomResourceDefinition
+				extClient     *extclient.Clientset
+				nfdClient     *nfdclient.Clientset
+				configMapName = "nfd-master-conf"
+				configMapKey  = "nfd-master.conf"
+				masterPodSpec []testpod.SpecOption
+			)
+
+			JustBeforeEach(func() {
```

Comment:
What I tried to express is that we don't need to repeat all the set-up tear down stuff that's already present in the test suite above. We just need to change [this](https://github.com/kubernetes-sigs/node-feature-discovery/blob/13f92faa771d58c15b40e92543cdef8bd711021e/test/e2e/node_feature_discovery_test.go#L206) to `JustBeforeEach` and adjust the tests with a `BeforeEach` to create the nfd-master pod with correct options.

Move this test to the suite above and we can drop basically all the BeforeAll, AfterAll, JustBeforeEach etc.